js/models/Settings.js

```diff
@@ -23,13 +23,21 @@ export default class extends BaseModel {
     return app.getServerUrl('ob/settings/');
   }
 
-  nested() {
+  get nested() {
```

Comment:
It shouldn't and doesn't when I try:

![image](https://cloud.githubusercontent.com/assets/794517/20935238/c175772a-bb92-11e6-91e1-e3a1c07b4ac6.png)